auth/src/main/java/com/firebase/ui/auth/ErrorCodes.java

```diff
@@ -0,0 +1,20 @@
+package com.firebase.ui.auth;
+
+/**
+ * Error codes retrieved from {@link IdpResponse#getErrorCode()}.
+ */
+public final class ErrorCodes {
+    private ErrorCodes() {
+        // We don't want people to initialize this class
+    }
+
+    /**
+     * Sign in failed due to lack of network connection
+     **/
+    public static final int NO_NETWORK = 10;
```

Comment:
Ah ok i see what you did here.  So you're returning it in both places for SignInDelegate, with the recommendation being that from here on out people use the errorCode to make decisions when the result is not OK.  That makes sense!